morphic.js

```diff
@@ -4496,6 +4496,56 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
+    
+    // Add hidden text box for copying and pasting
+    this.hiddenText = document.createElement('textarea');
+    this.target.hiddenText = this.hiddenText;
```

Comment:
I'm not sure, since to be honest. I'll try removing it and see what happens. :-)

Though, may @cyderize knows, since he really deserves the credit for this!